.gitpod.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-full
+
+ENV PATH /home/linuxbrew/.linuxbrew/bin:$PATH
```

Comment:
IMO, there is no need to explicitly append brew bin path. See here: <https://github.com/gitpod-io/workspace-images/blob/97e963a6b7d91ef2eb6e03d65f2de488297865a5/full/Dockerfile#L68>.

Reply:
Yes, cool.